ui/apps/platform/src/services/MetadataService.ts

```diff
@@ -18,24 +18,31 @@ export function fetchMetadata(): Promise<{ response: Metadata }> {
 // Provides availability of certain functionality of Central Services in the current configuration.
 // The initial intended use is to disable certain functionality that does not make sense in the Cloud Service context.
 
-// "Unknown" means "not disabled".
-// The reason it's not called "Enabled" is that extended checks might be required to confirm true ability to use
-// e.g. container AWS IAM role or GCP workload identity, and we can't positively say "Enabled" while such checks
-// aren't implemented.
-// This means the user should be allowed to use the capability (both via UI and API) but an attempt may not be
-// successful if the corresponding configuration does not match the actual environment.
-
-// Capability is disabled, meaning the corresponding UI should be disabled and attempts to use related APIs
+// CapabilityAvailable means that UI and APIs should be available for users to use.
+// This does not automatically mean that the functionality is 100% available and any calls to APIs will result
+// in successful execution. Rather it means that users should be allowed to leverage the functionality as
+// opposed to CapabilityDisabled when functionality should be blocked.
+
+// CapabilityDisabled means the corresponding UI should be disabled and attempts to use related APIs
 // should lead to errors.
 
-export type CentralServicesCapabilityStatus = 'Unknown' | 'Disabled';
+export type CentralServicesCapabilityStatus = 'CapabilityAvailable' | 'CapabilityDisabled';
```

Comment:
@pedrottimark I renamed this and other attributes in the contract, but at the current state it seems this functionality isn't yet used in UI so I hope should be fine.

Reply:
Yes, thank you for keeping backend and frontend consistent.

For your info, conditional rendering in code:
* will depend on property keys, for example: `getCentralServicesCapabilityIsDisabled('centralCanDisplayDeclarativeConfigHealth')`
* but not status constants, because helper function encapsulates them